cobigen-eclipse/cobigen-eclipse-test/src/main/resources/UpgradeMonolithicTemplatesTest/input/test.yml

```diff
@@ -0,0 +1,109 @@
+openapi: 3.0.0
+servers:
+  - url: 'https://localhost:8081/server/services/rest'
+    description: Just some data
+info:
+  title: Devon Example
+  description: Example of a API definition
+  version: 1.0.0
+  x-rootpackage: com.devonfw.angular.test
+paths:
+  /shopmanagement/v1/shop/{shopId}:
```

Comment:
done

Reply:
I'm still not sure about using an OpenAPI file as input. Better would be a very simple Java class I think.

Reply:
This test needs a very specific input to at atleast have one matching trigger from the templates. A java class will not be enough since there are no matching triggers in that case.